backend/src/nodes/nodes/utility/random_number.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+from typing import Union
+from random import randint, Random

```

Comment:
Lint fails because `randint` is an unused import.
```suggestion
from random import Random

```